staging/cse/windows/configfunc.tests.ps1

```diff
@@ -22,7 +22,7 @@ Describe 'Adjust-DynamicPortRange' {
             $global:EnableIncreaseDynamicPortRange = $false

```

Comment:
I think we also need to update the condition from "Should call Invoke-Executable 1 times" to "Should call Invoke-Executable 2 times"

Reply:
yes. it was caught by ut and I have fixed it.